search-sentiments-map/routes/search.js

```diff
@@ -390,7 +390,7 @@ async function addWorldDataByTopicToDatastore(topic, countriesData) {
     await datastore.save(entity);
     console.log(`Custom Search Result ${worldDataByTopicKey.id} created successfully.`);
   } catch (err) {
-    console.error('ERROR: ', err);
+    console.error('ERROR:', err);
```

Comment:
Add back the space. Not sure why it was removed. I am pretty sure it was originally added by request of a comment from Haylee.

Reply:
? But when you have a comma it automatically puts a space

Reply:
Oh I didn't notice the comma. Got it.